tests/_support/AcceptanceTester.php

```diff
@@ -345,6 +345,11 @@ public function deactivateWooCommerceSubscriptions() {
     $i->cli(['plugin', 'deactivate', self::WOO_COMMERCE_SUBSCRIPTIONS_PLUGIN]);
   }
 
+  public function isPluginActive(string $plugin): bool {
```

Comment:
I think it would be nicer to use a name like `checkPluginIsActive` so that it reads better when used with the tester variable ($i). `$i->isPluginActive` vs `$i->checkPluginIsActive`. What do you think?